frontend/src/views/WheresMyGene/components/HeatMap/index.tsx

```diff
@@ -161,10 +158,8 @@ export default memo(function HeatMap({
           <XAxisMask />
           <XAxisChart geneNames={sortedGeneNames} />
         </XAxisWrapper>
-        <FlexRow>
-          <YAxisWrapper
-            height={(chartWrapperRect?.height || 0) - X_AXIS_CHART_HEIGHT_PX}
-          >
+        <InlineRow>
```

Comment:
I think we can just use div since by default it's inline! Or maybe we no longer need it at all?

Reply:
True! It's not needed period.